e2e-tests/custom-login/specs/custom-login-flow-spec.js

```diff
@@ -37,8 +37,11 @@ describe('Custom Login Flow', () => {
     loginHomePage.clickLoginButton();
     customSignInPage.waitForPageLoad();
 
-    // Verify that curent domain hasn't changed to okta-hosted login, rather a local custom login page
-    expect(browser.getCurrentUrl()).not.toContain('okta');
+    // Verify that current domain hasn't changed to okta-hosted login, rather a local custom login page
+    const currentUrl = await browser.getCurrentUrl();
+    const isTrexOrg = currentUrl.includes('trex');
```

Comment:
We might want to make this more generic, as the URL could be from a custom domain.
Maybe grab the host from the configured issuer env var?

Reply:
Makes sense. Have updated the PR to check for issuer hostname instead.